Resources/Private/Partials/Errors/BackendLayoutNotSupported.html

```diff
@@ -13,7 +13,7 @@ <h4 class="alert-title">
 					You previously selected "{layout}" as backend layout, which is not supported by the currently selected theme.
 				</p>
 				<p>
-					Please select another one in the properties of your page.
+					Please select another one in the properties of your page. If you created your own backendlayout and you are sure to have selected the right one, please verify that the identifier in your backend-configuration reads exactly (capitalization inclusive!) like the filename of your theme-template.
```

Comment:
If you use your own backend-layout, please make sure that you have selected the correct one. 
Verify that the identifier in your backend-configuration matches the filename of your theme-template. (case sensitive).